README.md

```diff
@@ -25,6 +25,7 @@ grim -o DP-1 screenshot.png # Screenshoot a specific output
 grim -g "10,20 300x400" screenshot.png # Screenshoot a region
 slurp | grim -g - screenshot.png # Select a region and screenshoot it
 grim $(xdg-user-dir PICTURES)/$(date +'%Y-%m-%d-%H%M%S_grim.png') # Use a timestamped filename
+grim -o $(swaymsg -t get_outputs | jq -r '.[] | select(.focused==true) | .name') /tmp/screenshot.png # Grab a screenshot from the focused monitor
```

Comment:
`select(.focused)` works too

Is it possible to put the comment on the line before the command, since it's a little long? Also set the output file to just `screenshot.png`, like the other examples.

Reply:
Made the suggested changes.